src/components/MyEvents.js

```diff
@@ -345,6 +400,7 @@ const MyEvents = () => {
                     <CardActions>
                         {event.status === "Active" && <Button onClick={() => handleOpenDialog(event)}>Edit Event</Button>}
                         {event.status === "Active" && <Button onClick={() => handleOpenCancelDialog(event)}>Cancel Event</Button>}
+                        {event.status === "Active" && <Button onClick={() => handleOpenParticipantsDialog(event)}>See Participants</Button>}
```

Comment:
It might make more sense not to do conditional rendering here: let's say a user has a cancelled event but they would like to see who signed up. Thoughts?

Reply:
Yeah thats a good point, I'll remove the conditional.